sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1438,7 +1438,7 @@ object CleanupAliases extends Rule[LogicalPlan] {
   def trimNonTopLevelAliases(e: Expression): Expression = e match {
     case a: Alias =>
       Alias(trimAliases(a.child), a.name)(
-        a.exprId, a.qualifiers, a.explicitMetadata, a.isGenerated)
+        a.exprId, a.qualifier, a.explicitMetadata, a.isGenerated)
```

Comment:
not related to this PR, but we can simplify this to: `a.withNewChildren(trimAliases(a.child) :: Nil)`